py/desispec/workflow/procfuncs.py

```diff
@@ -96,7 +96,9 @@ def check_for_outputs_on_disk(prow, resubmit_partial_complete=True):
             'stdstarfit': 'stdstars',
             'poststdstar': 'cframe',
             'nightlybias': 'biasnight',
+            'zero': 'biasnight',
```

Comment:
Should this be `'bias': 'biasnight'`?

Reply:
good question.  It turns out this was an extraneous addition.  The jobdesc I intended to cover was already the `'nightlybias': 'biasnight'` entry.  In the end `bias` is a special-case obstype (meaning process all of the zeros into a nightly bias), and `nightlybias` (no dark) or `ccdcalib` (with dark) are the resulting jobdesc entries.  I removed this line and re-ran all the tests to confirm it didn't break anything.